opennlp-tools/src/main/java/opennlp/tools/formats/brat/BratDocumentParser.java

```diff
@@ -112,23 +114,42 @@ public BratDocumentParser(SentenceDetector sentenceDetector, Tokenizer tokenizer
         if (ann instanceof SpanAnnotation) {
           SpanAnnotation entity = (SpanAnnotation) ann;
 
-          Span entitySpan = entity.getSpan();
+          // TODO: create a list of token spans
+          List<Span> mappedFragments = new ArrayList<>();
 
-          if (sentence.contains(entitySpan)) {
-            entityIdSet.remove(ann.getId());
+          for (Span entitySpan : entity.getSpans()) {
+            if (sentence.contains(entitySpan)) {
+              entityIdSet.remove(ann.getId());
 
-            entitySpan = entitySpan.trim(sample.getText());
+              entitySpan = entitySpan.trim(sample.getText());
 
```

Comment:
nitpick: ident here and the 2 lines below

Reply:
Indents are good here, no?